code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -70,7 +70,7 @@
                 }
                 Set<Identity> identities = getIdentities(dn);
                 Token token = getTokenUtils().createToken(identities, null);
-                authToken = authTokenDao.createToken(token.getJwt(), getConstantsConfig().getConfig(), account.getId());
+                authToken = authTokenDao.createToken(token.getJwt(), getConstantsConfig().getConfig(), account.getId(), account.getId());
             }
```

Comment:
At this point above, is the account_id column in the auth_token db set to the "authenticatedAsAccountId" value instead of the account of kind "token"?